galaxy/values.yaml

```diff
@@ -166,7 +166,7 @@ ingress:
   hosts:
     - host: ~
       paths:
-        - path: "/"
```

Comment:
Why this change?

Reply:
Because on line 165 above it's always been set to default to `/galaxy` and the two paths should not mismatch.